Makefile

```diff
@@ -55,10 +55,6 @@ ifneq (,$(findstring plus,$(MAKECMDGOALS)))
    PLUS_ENABLED = true
 endif
 
-ifeq ($(CI),true)
-	GITHUB_OUTPUT := --github-output
-endif
-
```

Comment:
We still need this for the tests, maybe we can move it in that Makefile

Reply:
I think this is only a flag for ginkgo.

Reply:
Oh nevermind, I misread your comment.